index.html

```diff
@@ -56,6 +57,264 @@ <h2 class="connect">LET'S CONNECT</h2>
+                    <li><img src="images/Counter.png-2.png" alt="circle" /></li>
+                    <li>2015</li>
+                  </ul>
+                </div>
+
+              <div class="primary-text">
+                <p>
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required.
+                </p>
+              </div>
+              <div class="languages">
+                <ul class="programming-languages">
+                  <li><a href="#" class="btn">html</a></li>
+                  <li><a href="#" class="btn">css</a></li>
+                  <li><a href="#" class="btn">JavScript</a></li>
+                </ul>
+              </div>
+              <div class="text-see-project">
+                <a class="see-project-button" href="#">See Project</a>
+              </div>
+            </div>
+          </div>
+
+          <div class="project-card">
+            <a href="#snapshot">
+              <img
+              class="snapshot-portfolio-img"
+                src="images/Snapshoot Printing.png"snapshot-portfolio"
+              />
+            </a>
+
+            <div class="left-block">
+              <h2 class="project-title">Multi-Post
+                Stories</h2>
+                <div class="counter">
+                  <h3 class="canopy">CANOPY</h3>
+                  <ul class="canopy-text">
+                    <li><img src="images/Counter.png" alt="circle" /></li>
+                    <li>Back End Dev</li>
+                    <li><img src="images/Counter.png-2.png" alt="circle" /></li>
+                    <li>2015</li>
+                  </ul>
+                </div>
+
+              <div class="primary-text">
+                <p>
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required.
+                </p>
+              </div>
+              <div class="languages">
+                <ul class="programming-languages">
+                  <li><a href="#" class="btn">html</a></li>
+                  <li><a href="#" class="btn">css</a></li>
+                  <li><a href="#" class="btn">JavScript</a></li>
+                </ul>
+              </div>
+              <div class="text-see-project">
+                <a class="see-project-button" href="#">See Project</a>
+              </div>
+            </div>
+        </div>
+
+          
+      <section id="about-info" class="about-myself-section">
+        <div class="top">
+          <div class="title">
+            <h2 class="info-title">About Myself</h2>
+          </div>
+          <div class="detailed">
+            <p>
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design :relaxed: 
- [ ] The `About Myself` should take two lines. You can set the maximum width property for it to break the Myself to the next line.
- [ ] The about myself section is not in a form of a card as the project card. :smile: 

| Your Design |Figma Design |
| :---:   | :---: | 
|![Screenshot from 2022-12-06 16-40-40](https://user-images.githubusercontent.com/92303891/205970791-477a5562-646c-44b9-b278-bca2be14feb4.png)| ![Screenshot from 2022-12-06 16-33-19](https://user-images.githubusercontent.com/92303891/205970308-320780aa-4f53-4f8c-bec3-c57cc181b873.png)|